src/main/java/org/madhuri/app/message/dao/ChannelDAO.java

```diff
@@ -87,11 +82,21 @@ public Channel updateChannel(Channel existingChannel) {
 		return existingChannel;
 	}
 
-	public Channel getChannel(Long channelId) {
+	public Channel getChannelById(long id) {
 		Session session = HibernateUtil.getSession();
+		Channel channel = session.get(Channel.class, id);
+		session.close();
+		return channel;
+	}
+	
+	
+
+	public Channel getChannelById(Long channelId) {
```

Comment:
how is this different from getChannel ? why do we have 2 functions doing the same thing ?